nimble/controller/src/ble_ll_scan.c

```diff
@@ -3100,7 +3051,8 @@ ble_ll_scan_rx_pkt_in(uint8_t ptype, struct os_mbuf *om, struct ble_mbuf_hdr *hd
             if (BLE_MBUF_HDR_WAIT_AUX(hdr)) {
                 hdr->rxinfo.flags &= ~BLE_MBUF_HDR_F_AUX_PTR_WAIT;
                 if (ble_ll_sched_rmv_elem(&aux_data->sch) == 0) {
-                    ble_ll_scan_aux_data_unref(aux_data);
+                    ble_ll_scan_aux_data_unref(aux_data->sch.cb_arg);
```

Comment:
this looks odd, what is being unrefed here?

Reply:
I've pushed changed code which now have comment about this.  Basically, in here we remove scheduled item which contains its `aux_data` as a `cb_arg` and this is what we want to unref.

Reply:
but scheduler in aux_data should point to the same aux_data...

Reply:
ok, now I get it (it just explicitly states which ref in urefed)